### ISSUE TITLE
Cannot connect to sg and jp

### ISSUE THREAD

**Issue by CasanierRC** [ragibkl/adblock-dns-server#206]
Are sg and jp disconnected now?
But there is no problem with fr and us.

**Comment by ragibkl** [comment]
https://status.bancuh.com/

Status looks fine.
Also I tested them just now with dot protocol.

Should be working.

What kind of issues are you facing?

**Comment by CasanierRC** [comment]
I tried changing my IP using a VPN and it worked fine.
So it's most likely a problem with my ISP.
In the past, my ISP would sometimes fail to connect to certain websites...
Anyway, sorry and thank you!

**Comment by ragibkl** [comment]
Hmm. That's weird. I wonder if your isp purposely block our dns server ip? Or maybe they block ports for dns and dot? Does your device support doh protocol?

**Comment by CasanierRC** [comment]
I use browsers that support DoH.
When I switched to US and FR it worked fine, so it shouldn't be that DoH is blocked.
Also I couldn't connect to the JP and SG log pages yesterday.
I remember that the PHP website was blocked once before, but I was able to connect to it with a VPN.

But today I was able to connect to JP and SG and everything is back to normal.
It's still hard to understand why this is happening, but I can only assume that it's a problem with the telecom provider on my end.
Translated with DeepL.com (free version)
